src/modules/collections/services/collections.service.ts

```diff
@@ -25,15 +25,21 @@ import {
 	RemoveObjectFromCollectionDocument,
 	UpdateCollectionDocument,
 } from '~generated/graphql-db-types-hetarchief';
+import { PlayerTicketService } from '~modules/admin/player-ticket/services/player-ticket.service';
 import { CollectionObjectsQueryDto } from '~modules/collections/dto/collections.dto';
 import { DataService } from '~modules/data/services/data.service';
 import { PaginationHelper } from '~shared/helpers/pagination';
 
 @Injectable()
 export class CollectionsService {
 	private logger: Logger = new Logger(CollectionsService.name, { timestamp: true });
+	// The current referer header
+	protected referer: string;
```

Comment:
Why are you storing the referer inside a service? Shouldn't that be on a "per request" bases instead of across different requests?

Reply:
fixed